### PR TITLE
Heartbeat rewrite

### PR DESCRIPTION
**Pull request by johan-bjareholt**
Rewrote the watcher with the heartbeat API

Had to rewrite a lot of the logic so wanted to test the code for a longer period of time before making a PR, but I have tested it for about a week now and it seems to be working well.

Fixes https://github.com/ActivityWatch/aw-watcher-afk/issues/10

### CODE REVIEW

**Comment by johan-bjareholt** [comment]
@ErikBjare Works well for me, review plz?

**Comment by ErikBjare** [comment]
Was just about to ask :)

File: aw_watcher_afk/unix.py [hunk 1]
```diff
@@ -37,17 +37,17 @@ def time_since_last_input(self) -> float:
 _last_input_unix = None
 
 
-def time_since_last_input():
+def seconds_since_last_input():
     global _last_input_unix
```
Comment (ErikBjare) [inline aw_watcher_afk/unix.py]: I know that you didn't write this, but it'd be better to attach the LastInputUnix object to the function object.

```python
def seconds_since_last_input():
    if not hasattr(seconds_since_last_input, "last_input_unix"):
        seconds_since_last_input.last_input_unix = LastInputUnix()
    return seconds_since_last_input.last_input_unix.seconds_since_last_input()
```

This way there is no global.

**Comment by ErikBjare** [comment]
LGTM!

File: aw_watcher_afk/unix.py (hunk 1 above)
Comment (johan-bjareholt) [inline aw_watcher_afk/unix.py]: Feel free to commit that change if you want before i merge

Comment (ErikBjare) [inline aw_watcher_afk/unix.py]: Nah, it's cool. Just merge.

**Pull request merged**